### ISSUE TITLE
Move oz_tree_build/tree_build/token_to_oz_tree_file_mapping.py to tree directory?

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/tree-build#52]
The file `token_to_oz_tree_file_mapping.py` is specific to the AllLife tree in `data/OZTreeBuild/`. So it probably belongs in `data/OZTreeBuild/AllLife/`?

**Comment by davidebbo** [comment]
Indeed, it would conceptually belong there. Though I worry that putting code in the data folder is a bit quirky. The import line from `build_oz_tree.py` currently looks like this (same folder import):

```python
from .token_to_oz_tree_file_mapping import token_to_file_map
```

If it were in `data/OZTreeBuild/AllLife/`, we'd have to do some import gymnastic.

It's worth noting that this very concept of having those mappings may not be used outside of the main tree. e.g. for the extinct case based on wikipedia, hopefully we can stay away from having bespoke trees saved in the repo.

Long story short, while a bit quirky, this may not be worth changing at this time.
